### PR TITLE
feat: bump cosmwasm version #LSC-315

### PR DESCRIPTION
**Pull request by ratik**
Updated cosmwasm version and dependent stuff

### CODE REVIEW

File: contracts/lido_cosmos_hub/src/contract.rs [hunk 1]
```diff
@@ -388,9 +386,10 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
 
 fn query_guardians(deps: Deps) -> StdResult<Vec<String>> {
     let guardians = GUARDIANS.keys(deps.storage, None, None, Order::Ascending);
-    let guardians_decoded: Result<Vec<String>, FromUtf8Error> =
-        guardians.map(String::from_utf8).collect();
-    Ok(guardians_decoded?)
+
+    let guardians_decoded: Result<Vec<String>, StdError> = guardians.collect();
+
+    Ok(guardians_decoded.unwrap())
```
Comment (zavgorodnii) [inline contracts/lido_cosmos_hub/src/contract.rs]: i guess we don't like unwraps?

File: contracts/lido_cosmos_hub/src/testing/mock_querier.rs [hunk 2]
```diff
@@ -63,6 +64,7 @@ pub fn mock_dependencies(
         storage: MockStorage::default(),
         api: MockApi::default(),
         querier: custom_querier,
+        custom_query_type: PhantomData,
```
Comment (zavgorodnii) [inline contracts/lido_cosmos_hub/src/testing/mock_querier.rs:67]: 👻

File: contracts/lido_cosmos_hub/src/contract.rs (hunk 1 above)
Comment (ratik) [reply contracts/lido_cosmos_hub/src/contract.rs]: oh got it. I'm not sure about the current solution, could you please recheck

**Review by swelf19 — APPROVED** [review]

File: contracts/lido_cosmos_hub/src/testing/mock_querier.rs (hunk 2 above)
Comment (foxpy) [reply contracts/lido_cosmos_hub/src/testing/mock_querier.rs:67]: Well, cosmwasm-std uses that, I guess we are forced to follow their decision.

**Review by foxpy — APPROVED** [review]

File: contracts/lido_cosmos_hub/src/contract.rs (hunk 1 above)
Comment (foxpy) [reply contracts/lido_cosmos_hub/src/contract.rs]: > I'm not sure about the current solution

Looks good for my taste.

**Comment by zavgorodnii** [comment]
this looks good. is it okay if we keep this open for some time until you successfully run your first integration test?

**Pull request merged**